website/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Triphouse project</title>
+</head>
+<body>
+    <div class="container">
+        <div class="first_block">
+            <h1>What do we offer</h1>
+            <div class="line"></div>
+        </div> 
+        <div class="item_container">
+            <div class="item">
+                <img class="item_icon" src="img/Support.png">
+                <div class="item_title">Support 24/7</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Communicate.png">
+                <div class="item_title">Communicate directly</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Book.png">
+                <div class="item_title">Book online</div> 
+            </div>
+            <div class="item">
+                <img class="item_icon" src="img/Real guest.png">
+                <div class="item_title">Real guest reviews</div> 
+            </div>
+        </div>
+        <div class="footer">
+            <ul class="footer_column">
+                <li class="footer_title">About</li>
+                <li class="footer_text">How Triphouse works</li>
+                <li class="footer_text">Careers</li>
+                <li class="footer_text">Privacy</li>
+                <li class="footer_text">Terms</li>
+            </ul>
+            <ul class="footer_column">
+                <li class="footer_title">Property types</li>
+                <li class="footer_text">Guest houses</li>
+                <li class="footer_text">Hotels</li>
+                <li class="footer_text">Apartments</li>
+                <li class="footer_text">Villas</li>
+                <li class="footer_text">Holiday homes</li>
+                <li class="footer_text">Hostels</li>    
+            </ul>
+            <ul class="footer_column">
+                <li class="footer_title">Support</li>
+                <li class="footer_text">Contact Customer Service</li>
+                <li class="footer_text">FAQ</li>
+            </ul>
+        </div>
+        <div class="copyright">
+            <div class="copyright_text">&#169; 2020 Triphouse, Inc. All rights reserved</div>
+        </div>
```

Comment:
you can put in inside footer tag for avoiding background-color styles. this will be more semantically correct